src/main/java/ai/knowly/langtorch/parser/textsplitter/CharacterTextSplitter.java

```diff
@@ -0,0 +1,30 @@
+package ai.knowly.langtorch.parser.textsplitter;
```

Comment:
ai.knowly.langtorch.preprocessing.splitter.text?

Could you also move the parser into this preprocessing? I feel like we probably have image, csv, etc stuff. Like pdf for pdf chatbot very soon.